internal/console/assets/scan-flags.json

```diff
@@ -195,5 +195,11 @@
     "shorthandFlag": "",
     "defaultValue": "false",
     "usage": "disables the exclusion of paths specified within .gitignore file"
+  },
+  "max-file-size": {
+    "flagType": "int",
+    "shorthandFlag": "",
+    "defaultValue": "5",
+    "usage": "max file size that allowed to scan in MB"
```

Comment:
the description contains an error. I would advise changing it to "max scannable file size, in MB" or "max file size that will be scanned, in MB" or "max file size that is allowed to be scanned, in MB"